lib/app/widgets/custom_textfield.dart

```diff
@@ -53,11 +57,7 @@ class _CustomTextFieldState extends State<CustomTextField> {
   @override
   Widget build(BuildContext context) => Row(
         children: [
-          Icon(
-            widget.iconData,
-            size: widget.iconData != null ? AppDimensions.d50 : 0,
-            color: AppColors.daintree,
-          ),
+          widget.leading ?? const SizedBox.shrink(),
```

Comment:
Try:

```suggestion
if(widget.leading != null)
          widget.leading,
```

SizedBox is not going to be rendered if you add it like this